### PR TITLE
DOCSP-46048 First-Round Edits: Backups Page

### PR DESCRIPTION
**Pull request by kanchana-mongodb**
- [JIRA](https://jira.mongodb.org/browse/DOCSP-46048)
- [Stage](https://deploy-preview-66--docs-atlas-architecture.netlify.app/backups/)

### CODE REVIEW

File: source/backups.txt [hunk 1]
```diff
@@ -224,26 +221,30 @@ data loss during failures. |service| can quickly recover to the exact
...
-You can also use incremental backups and
-built-in compression to minimize the amount of stored data. By selecting
-regions strategically for backup, you can avoid cross-region data
-transfer fees and choose the right cluster size based on workload to
-prevent overspending. By implementing these strategies, you can
-effectively manage costs while maintaining secure and reliable backups. 
+unnecessary storage expenses. For example, you should disable backups in
+lower environments, and ensure that in upper environments with high
+availability requirements you are distributing backups to each region
+where you are deployed. |service| provides incremental backups through
```
Comment (sarahsimpers) [inline source/backups.txt:240]:
```suggestion
where your |service| {+clusters+} are deployed. |service| provides incremental backups through
```

File: source/includes/examples/cli-example-backup-compliance-policy-schedule.rst [hunk 2]
```diff
@@ -3,7 +3,7 @@
 
    atlas backups compliancePolicy policies scheduled create \ 
      --projectId 67212db237c5766221eb6ad9 \
-     --frequencyInterval 2 \
```
Comment (sarahsimpers) [inline source/includes/examples/cli-example-backup-compliance-policy-schedule.rst:6]: Question: Do we need to update these for Terraform examples as well?

**Review by sarahsimpers — APPROVED** [review]
LGTM % a few small changes, and one question.

Successfully tested the Atlas CLI example:
Screenshot 2024-12-27 at 7 05 31 AM

File: source/includes/examples/cli-example-backup-compliance-policy-schedule.rst (hunk 2 above)
Comment (kanchana-mongodb) [reply source/includes/examples/cli-example-backup-compliance-policy-schedule.rst:6]: No, because we have different examples for Terraform.

**Pull request merged**